core/Convert.php

```diff
@@ -445,4 +445,42 @@ public static function base64url_decode($val) {
 			true
 		);
 	}
+
+	/**
+	 * Convert a php.ini value (eg: 512M) to bytes
+	 *
+	 * @todo unit tests
+	 *
+	 * @param string $iniValue
+	 * @return int
+	 */
+	public static function ini2bytes($iniValue) {
```

Comment:
I might just move them back to File dataobject then; They didn't hurt being there really.